.github/workflows/issue-labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: Issue Labeler
+on:
+  issues:
+    types: [opened, edited]
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: github/issue-labeler@v3.1 
+      with:
+        configuration-path: .github/issue-labeler.yml
+        not-before: 2020-01-15T02:54:32Z
+        enable-versioned-regex: 0
```

Comment:
```suggestion

```
We don't really need all this. I would suggest using a much simpler approach for this. We can simply add label using `github-script`

Reply:
Using the action sets up for additional label automation later if we want it.